packages/core/graph/src/Graph.js

```diff
@@ -34,17 +39,24 @@ export default class Graph<TNode, TEdgeType: number = 1> {
     this.setRootNodeId(opts?.rootNodeId);
 
     let adjacencyList = opts?.adjacencyList;
-    this.adjacencyList = adjacencyList
-      ? AdjacencyList.deserialize(adjacencyList)
-      : new AdjacencyList<TEdgeType>();
+
+    if (adjacencyList) {
+      this.adjacencyList = AdjacencyList.deserialize(adjacencyList);
+    } else {
+      // default edgeTypes to array containing just the null edge type
+      let edgeTypes = new Uint8Array(
+        opts?.edgeTypes ? objectValues(opts.edgeTypes) : [1],
```

Comment:
For now I have defaulted `edgeTypes` to just the null edge type. However, I think this is probably a bad idea as it could easily lead to mistakes if more graphs are created. We should probably make it a required property. This will lead to a larger change though as it will mean passing it to all constructors, including tests. 

Thoughts?